api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -204,6 +211,7 @@ message HttpHeaders {
   config.core.v3.HeaderMap headers = 1;
 
   // [#not-implemented-hide:]
+  // TODO(jbohanon) reserve field as part of rework detailed in https://github.com/envoyproxy/envoy/issues/32125
```

Comment:
Is this TODO necessary? If the `attributes` in the `ProcessingRequet` should be used, then please mark this field as deprecated directly.

Reply:
TODO will be removed at time of implementation